### ISSUE TITLE
Add a section for Recommendations/References

### ISSUE THREAD

**Issue by amoghjalan** [CatsInTech/Rezume#40]
Recommendations is an essential part of a Resume.
One can add  references of people who have had enough direct experience with you to testify to your character, conduct and competencies.

It Should have following fields:

- [ ] Reference's Full Name

- [ ] Job Title

- [ ] Address of Organisation

- [ ] Contact Telefone

- [ ] Contact E-Mail Address

**Comment by PrathibhaS** [comment]
Do we need Address of Organisation. Isn't Name of Organisation sufficient?

**Comment by praveenscience** [comment]
@amoghjalan Please answer the above question and also we will add this to #8. Okay?

**Comment by praveenscience** [comment]
@PrathibhaS Just checked my references, and they do have address. So yea, the issue seems alright to me.

**Comment by amoghjalan** [comment]
> Do we need Address of Organisation. Isn't Name of Organisation sufficient?

It would be much like:
Google Inc.
Banglore, India.
It gives actually tells that where you worked in that organisation

**Comment by amoghjalan** [comment]
> @amoghjalan Please answer the above question and also we will add this to #8. Okay?

Yup

**Comment by shubkaushik** [comment]
Hi @praveenscience, if this is not taken, can i work on this ?

**Comment by SonikeshKumar14** [comment]
Hi! I would like to work on this issue under GSSOC`22. Please assign it to me.

**Comment by praveenscience** [comment]
@SonikeshKumar14 What do you mean work on this? Oh, review? Okay review and let me know what you find.

**Comment by sceary-expert** [comment]
I WANT TO WORK ON THIS ISSUE UNDER GSSOC'22.

**Comment by praveenscience** [comment]
OKay, sure, please review the **already completed** work, @sceary-expert, assigned to you and @SonikeshKumar14.

**Comment by praveenscience** [comment]
@SonikeshKumar14 Please let me know your reviews in the next few hours, else you'll get unassigned.

**Comment by sceary-expert** [comment]
Yes I will work on that. Can you give more description of the issue?

**Comment by praveenscience** [comment]
See the first comment, it's all clear. @sceary-expert.

**Comment by sulogna2001** [comment]
Hello can you assign me this issue under GSSOC 22

**Comment by praveenscience** [comment]
@sulogna2001 What are your plans about this? 😊  Please tell me?

**Comment by praveenscience** [comment]
@ShivangiChy Awesome one. Currently some PRs are pending for the same, I just need to merge them. But I am still unsure if recommendation part has been done. Can you check out the open PRs right now to see if it's done?

**Comment by ShivangiChy** [comment]
Hello!
Even I could not find any PRs related to Recommendation. So, shall I start working on this under GSSOC 22?

**Comment by praveenscience** [comment]
@ShivangiChy Any updates?
